Gemfile

```diff
@@ -46,7 +48,7 @@ source 'https://rubygems.org' do
   gem 'rails_autolink', '~> 1.1.6'
   # datetime picker
   gem 'momentjs-rails', '>= 2.9.0'
-  gem 'bootstrap3-datetimepicker-rails', '~> 4.15.35'
+  gem 'bootstrap4-datetime-picker-rails'
```

Comment:
think it is still better to lock to some version number or now

Reply:
I go check.

Reply:
Done